### DISCUSSION TITLE
Hardware recommendation for NUC

### DISCUSSION THREAD

**Discussion by alanpilz** [blakeblackshear/frigate#1397] [Ask A Question]
Hello.. I am running HA on a RPi4 with 4GB RAM + 480Gb SSD + Google Coral.

I would like to use Frigate with 7 CCTV cameras for person detection and have come to the conclusion that the RPi is just not up to the job. Adding just 3 cameras crashes HA.

What spec of NUC is recommended to get my all 7 CCTV cameras running with Coral. The cameras are 1920x1080 from Dahua connected via a NVR. Thanks

**Comment by tve** [comment]
I'm using a MinisForum GX41 (J4125 processor) plus USB EdgeTPU. So far very happy, but I have fewer cameras than you're planning to have. The box has one internal SATA port available so you could add a high capacity drive, depending on your recording plans.
